<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>919</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_12">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Device Info</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout" columnstretch="1,8">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Connected Devices:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBoxDevices"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>IN Endpoint:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxInEndPoints"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>OUT Endpoint:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBoxOutEndPoints"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Stream OUT / Stream IN (For Maxmum Speed Test)</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEditOutBytes"/>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="lineEditOutFailures"/>
             </item>
             <item row="4" column="2">
              <widget class="QLineEdit" name="lineEditOutAsyncFailures"/>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="lineEditOutAsyncBytes"/>
             </item>
             <item row="2" column="2">
              <widget class="QLineEdit" name="lineEditOutAsyncRate"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>OutBytes(KB)</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLineEdit" name="lineEditOutAsyncSuccesses"/>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Sync</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>OutRate(KB/s)</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEditOutRate"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>OutSuccesses</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>OutFailures</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="lineEditOutSuccesses"/>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Async</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>OUT</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButtonOutSyncContinue">
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>OutSyncContinue</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>OutAsyncContinue</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEditOutSyncOnceBytes"/>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="pushButtonOutSyncOnce">
               <property name="text">
                <string>OutSyncOnce</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="pushButtonOutAsyncContinue">
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QPushButton" name="pushButtonOutTxtClear">
               <property name="text">
                <string>Clear</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="plainTextEditBufferOut"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEditInRate"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>InBytes(KB)</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEditInBytes"/>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="lineEditInFailures"/>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="lineEditInSuccesses"/>
             </item>
             <item row="3" column="2">
              <widget class="QLineEdit" name="lineEditInAsyncSuccesses"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>InRate(KB/s)</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLineEdit" name="lineEditInAsyncRate"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>InSuccesses</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>InFailures</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="lineEditInAsyncBytes"/>
             </item>
             <item row="4" column="2">
              <widget class="QLineEdit" name="lineEditInAsyncFailures"/>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Sync</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Async</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_25">
               <property name="text">
                <string>IN</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>InSyncContinue</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEditInSyncOnceBytes"/>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="pushButtonInSyncOnce">
               <property name="text">
                <string>InSyncOnce</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButtonInSyncContinue">
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>InAsyncContinue</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="pushButtonInAsyncContinue">
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="pushButtonInTxtClear">
               <property name="text">
                <string>Clear</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="plainTextEditBufferIn"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Bulk Loop (For Loop CRC Test)</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>LoopSyncContinue</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonLoopSyncContinue">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_35">
             <property name="text">
              <string>tPcLoop(us)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditPcLoop"/>
           </item>
           <item>
            <widget class="QLabel" name="label_36">
             <property name="text">
              <string>tMaxPcLoop(us)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditMaxPcLoop"/>
           </item>
           <item>
            <widget class="QLabel" name="label_37">
             <property name="text">
              <string>tAvgPcLoop(us)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditAvgPcLoop"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEditLoopOutSuccesses"/>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>InFailures</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string>OutRate(KB/s)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>LoopBytes(KB)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QLabel" name="label_31">
             <property name="text">
              <string>InSuccesses</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLineEdit" name="lineEditLoopInSuccesses"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditLoopOutBytes"/>
           </item>
           <item row="1" column="5">
            <widget class="QLineEdit" name="lineEditLoopRate"/>
           </item>
           <item row="3" column="3">
            <widget class="QLineEdit" name="lineEditLoopInFailures"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditLoopOutRate"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>OutFailures</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEditLoopOutFailures"/>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_34">
             <property name="text">
              <string>InBytes(KB)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QLabel" name="label_32">
             <property name="text">
              <string>InFailures</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string>OutBytes(KB)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="lineEditLoopInBytes"/>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>InSuccesses</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="lineEditLoopInRate"/>
           </item>
           <item row="0" column="5">
            <widget class="QLineEdit" name="lineEditLoopBytes"/>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_33">
             <property name="text">
              <string>InRate(KB/s)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>LoopRate(KB/s)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QLineEdit" name="lineEditLoopSuccesses"/>
           </item>
           <item row="3" column="5">
            <widget class="QLineEdit" name="lineEditLoopFailures"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>OutSuccesses</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Bulk Plot(For Wave Plot)</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="lineEditWaveGeneInterval">
             <property name="text">
              <string>10</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_38">
             <property name="text">
              <string>(ms)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>WaveGene</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonShowWave">
             <property name="text">
              <string>ShowWave</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEditPlotOutSuccesses"/>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="label_39">
             <property name="text">
              <string>InFailures</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_40">
             <property name="text">
              <string>OutRate(KB/s)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QLabel" name="label_42">
             <property name="text">
              <string>PlotSuccesses</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLineEdit" name="lineEditPlotInSuccesses"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditPlotOutBytes"/>
           </item>
           <item row="1" column="5">
            <widget class="QLineEdit" name="lineEditPlotRate"/>
           </item>
           <item row="3" column="3">
            <widget class="QLineEdit" name="lineEditPlotInFailures"/>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditPlotOutRate"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_43">
             <property name="text">
              <string>OutFailures</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEditPlotOutFailures"/>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_44">
             <property name="text">
              <string>InBytes(KB)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QLabel" name="label_45">
             <property name="text">
              <string>PlotFailures</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_46">
             <property name="text">
              <string>OutBytes(KB)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="lineEditPlotInBytes"/>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_47">
             <property name="text">
              <string>InSuccesses</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="lineEditPlotInRate"/>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_48">
             <property name="text">
              <string>InRate(KB/s)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="label_49">
             <property name="text">
              <string>PlotRate(KB/s)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QLineEdit" name="lineEditPlotSuccesses"/>
           </item>
           <item row="3" column="5">
            <widget class="QLineEdit" name="lineEditPlotFailures"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_50">
             <property name="text">
              <string>OutSuccesses</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>743</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
